runtime/src/main/java/org/corfudb/runtime/Backup.java

```diff
@@ -54,6 +59,17 @@
     // All tables in Corfu Db
     private List<UUID> allTablesInDb;
 
+    // Metrics
+    private final Optional<DistributionSummary> backupTimeDistributionSummary =
+            MeterRegistryProvider.getInstance().map(registry ->
+                    DistributionSummary.builder("backup.table.time")
```

Comment:
Use timer for measuring times/latencies, also don't forget to include percentiles.
publishPercentiles(0.50, 0.99)
.publishPercentileHistogram()

Reply:
The appliance backup&restore team asks to log the times outside metrics log as well, so I have to explicitly take the timer and add it to both backup&restore log and metrics log (DistributionSummary).